src/status_im/components/tabs/views.cljs

```diff
@@ -9,20 +9,25 @@
             [status-im.components.tabs.styles :as tabs.styles]
             [status-im.utils.platform :as platform]))
 
-(defn- tab [{:keys [view-id title icon-active icon-inactive style-active selected-view-id on-press]}]
+(defn- tab [{:keys [view-id title title-fn icon-active icon-inactive style-active selected-view-id on-press]
+             :or {title-fn react/text}}]
   (let [active? (= view-id selected-view-id)
         text-only? (nil? icon-active)]
-    [react/touchable-highlight {:style    (merge tabs.styles/tab (if (and active? style-active) style-active))
+    [react/touchable-highlight {:style    (merge tabs.styles/tab
+                                                 (when (and active? style-active)
+                                                   style-active))
                                 :disabled active?
-                                :on-press  #(if on-press (on-press view-id) (dispatch [:navigate-to-tab view-id]))}
+                                :on-press #(if on-press
+                                             (on-press view-id)
+                                             (dispatch [:navigate-to-tab view-id]))}
      [react/view {:style tabs.styles/tab-container}
       (when-let [icon (if active? icon-active icon-inactive)]
         [react/view
          [vi/icon icon (tabs.styles/tab-icon active?)]])
-      [react/view
-       [react/text (merge (if text-only? {:uppercase? (get-in platform/platform-specific [:uppercase?])})
-                          {:style (tabs.styles/tab-title active? text-only?)})
-        title]]]]))
+      [react/view {:style tabs.styles/tab-title-wrapper}
+       (title-fn {:uppercase? (and text-only? (get-in platform/platform-specific [:uppercase?]))
```

Comment:
Not exactly the same here as `uppercase?` as no general meaning, only for text-only component.
Probably harmless.